drivers/scheduler/k8s/specs/kubevirt-fedora/vm-fedora.yaml

```diff
@@ -0,0 +1,169 @@
+            - disk:
+                bus: virtio
+              name: cloudinitdisk
+            {{ if .ClaimsCount }}
+            {{- range $val := Iterate .ClaimsCount }}
+            - name: datavolume{{ $val }}
+              disk:
+                bus: virtio
+            - disk:
+              name: app-config-disk-{{ $val }}
+              serial: CVLY623300HK240D{{ $val }}
+            - disk:
+              name: app-secret-disk-{{ $val }}
+              serial: D23YZ9W6WA5DJ487{{ $val }}
+            {{- end }}
+            {{ else }}
+            - name: datavolume
+              disk:
+                bus: virtio
+            - disk:
+              name: app-config-disk
+              serial: CVLY623300HK240D
+            - disk:
+              name: app-secret-disk
+              serial: D23YZ9W6WA5DJ487
+            {{- end }}
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 2048M
+      terminationGracePeriodSeconds: 0
+      volumes:
+        - name: containerdisk
+          containerDisk:
+            image: kubevirt/fedora-cloud-container-disk-demo:latest
+        {{ if .ClaimsCount }}
+        {{- range $val := Iterate .ClaimsCount }}
+        - name: datavolume{{ $val }}
+          dataVolume:
+            name: pvc-data-volume-{{ $val }}
+        - name: app-config-disk-{{ $val }}
+          configMap:
+            name: app-config-{{ $val }}
+        - name: app-secret-disk-{{ $val }}
+          secret:
+            secretName: app-secret-{{ $val }}
+        {{- end }}
+        {{ else }}
+        - name: datavolume
+          dataVolume:
+            name: pvc-data-volume
+        - name: app-config-disk
+          configMap:
+            name: app-config
+        - name: app-secret-disk
+          secret:
+            secretName: app-secret
+        {{- end }}
+        - cloudInitNoCloud:
+            userData: |-
+              #cloud-config
+              password: fedora
+              chpasswd: { expire: False }
+              bootcmd:
+                {{ if .ClaimsCount }}
+                {{- range $val := Iterate .ClaimsCount }}
+                - "sudo mkdir /mnt/app-config-{{ $val }}"
+                - "sudo mkdir /mnt/app-secret-{{ $val }}"
+                - "sudo mount /dev/$(lsblk --nodeps -no name,serial | grep CVLY623300HK240D{{ $val }} | cut -f1 -d' ') /mnt/app-config-{{ $val }}"
+                - "sudo mount /dev/$(lsblk --nodeps -no name,serial | grep D23YZ9W6WA5DJ487{{ $val }} | cut -f1 -d' ') /mnt/app-secret-{{ $val }}"
+                {{- end }}
+                {{ else }}
+                - "sudo mkdir /mnt/app-config"
+                - "sudo mkdir /mnt/app-secret"
+                - "sudo mount /dev/$(lsblk --nodeps -no name,serial | grep CVLY623300HK240D | cut -f1 -d' ') /mnt/app-config"
+                - "sudo mount /dev/$(lsblk --nodeps -no name,serial | grep D23YZ9W6WA5DJ487 | cut -f1 -d' ') /mnt/app-secret"
+                {{- end }}
```

Comment:
Can we have defaultClaimsCount?

Please see https://github.com/portworx/torpedo/pull/2291/files#diff-00dc274e79bb288d048e8f9d9fd387336b86b9415db60407d6999fbd4807e4f5R1

Reply:
How will this help?

Reply:
There will be no need of else.

Reply:
My bad. I thought I had assigned ClaimsCount a default value if it was not specified. I am not sure about the possibility. Please ignore.